utils/styleguide/public/images/garden.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="26" height="26" focusable="false" viewBox="0 0 26 26">
+  <path fill="#5EAE91" d="M2 13.5C2 19.302 6.921 24 13 24s11-4.698 11-10.5V3c-6.079 0-11 4.698-11 10.5C13 7.698 8.079 3 2 3v10.5z"/>
+</svg>
```

Comment:
Is this getting picked-up as the reference is coming from `.storybook`, not `styleguide`? Also, any options for sourcing this from the master SVG?

Reply:
We link this public assets directory using the `-s` command option in the npm tasks.

> any options for sourcing this from the master SVG

Their theming API doesn't allow us to supply a React element or anything that's styled. I modeled this directly from the `react-containers` repo where we use this same asset.